src/components/TodoList/TodoList.tsx

```
@@ -1,6 +1,17 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
+type Props = {
+  todos: Todo[];
+  onSelect: (todo: Todo) => void;
+  selectedTodo: Todo | null;
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  onSelect = () => {},
```

Issue: The onSelect prop is required in the Props type, so providing a default value (onSelect = () => {}) in the destructuring is unnecessary and could mask errors if the prop is not provided. Please remove the default value to ensure that the parent component always passes this prop.